### PR TITLE
Boilerplate

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
@rudigiesler ready for review

File: src/http.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  def = conj(def, {
+    headers: {},
+    paramsSerializer: stringifyQs
+  });
+
+  def = conj(def, {
+    url: opts.prefix + def.url,
+    headers: conj(def.headers, {Authorization: `Token ${opts.token}`})
+  });
```
Comment (rudigiesler) [inline src/http.js:20]: Is there a reason why we are not just bundling this with the previous conj (on line 16)?

**Comment by rudigiesler** [comment]
Just the one question from me, otherwise looks good 😄 . I'm liking the automatic fixtures for tests.

File: src/http.js (hunk 1 above)
Comment (justinvdm) [inline src/http.js:20]: Ah, this was for a reason that is not relevant anymore, will fix.

**Comment by justinvdm** [comment]
Ready for re-review.

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**